modules/oc-mirror-updating-use-cases.adoc

```diff
@@ -46,18 +45,60 @@ storageConfig:
 mirror:
   platform:
     channels:
-      - name: stable-4.13 #<1>
+      - name: stable-4.12
+        minVersion: 4.12.1
+        maxVersion: 4.12.1
+      - name: stable-4.14 
   operators:
     - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.14
       packages:
         - name: rhacs-operator
           channels:
           - name: stable
+            minVersion: 1.0.1
 ----
-<1> Replacing by `stable-4.13` prunes all the images of `stable-4.12`.
 
-[discrete]
-== Updating to the latest version of an Operator by pruning the existing images
+This image set configuration mirrors all the {product-title} releases necessary for the upgrade of the cluster from 4.12.1 to the latest version in channel `stable-4.14`.
+
+Add `stable-4.14` without removing 4.12.1 when you are still using 4.12.1. A rolling cluster reboot due to a `MachineConfiguration` change or other normal cluster operations might occur after you refresh the mirror but before you upgrade. You will need the images you are actively using.
+
+== Mirroring a specific {product-title} version and pruning the existing images
```

Comment:
Make sure all of these subsections in this module have unique IDs (per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files)). E.g.,:
```suggestion
[id="mirror-specific-version-prune-existing_{context}"]
== Mirroring a specific {product-title} version and pruning the existing images
```